perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -558,6 +572,21 @@ def Prepare(self, vm):
     logging.info('Successfully created an RDS DB instance. Address is %s',
                  vm.db_instance_address)
     logging.info('Complete output is:\n %s', response)
+     
+  def _validate_size(self, size):
+    """Validates flag for storage size, throws exception if invalid.
```

Comment:
Change to "Validate flag for storage size and throw exception if invalid." The style guide requests that summary doc strings are written as a command, not a description. Also, move  "AWS supports storage sizes from 1GB to 16TB currently."  Up so that it is one line below the summary and above the args and returns.